src/app/ApplicationTemplate.Tests.ApiClient/ApiEndpointTestBase.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true)]
+
+namespace ApplicationTemplate.Tests;
+
+/// <summary>
+///  Gives access to the API endpoints and their configuration.
+/// </summary>
+public abstract class ApiEndpointTestBase : IDisposable
```

Comment:
naming is a bit weird between this base class and the project name. Also I don't know what you be the correct name for tests that check the behavior of remote APIs. I'm not sure ApiClient is the right thing.

Reply:
I forgot to rename it. Nice catch!